vendor/github.com/Azure/go-ansiterm/LICENSE

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
I have not done the review yet, but can you please split all the vendor changes into a new commit called `go mod vendor`? 

Anyway, nice to see the deps going away.